### ISSUE TITLE
Issue while running the colab notebook on ploomber examples

### ISSUE THREAD

**Issue by idomic** [ploomber/ploomber#741]
```
File "/usr/local/lib/python3.7/dist-packages/ploomber/__init__.py", line 2, in <module>
    from ploomber.dag.dag import DAG
  File "/usr/local/lib/python3.7/dist-packages/ploomber/dag/dag.py", line 93, in <module>
    from ploomber.dag.dagclients import DAGClients
  File "/usr/local/lib/python3.7/dist-packages/ploomber/dag/dagclients.py", line 4, in <module>
    from ploomber.tasks.abc import Task
  File "/usr/local/lib/python3.7/dist-packages/ploomber/tasks/__init__.py", line 6, in <module>
    from ploomber.tasks.notebook import NotebookRunner, ScriptRunner
  File "/usr/local/lib/python3.7/dist-packages/ploomber/tasks/notebook.py", line 15, in <module>
    from nbconvert.exporters.webpdf import WebPDFExporter
ModuleNotFoundError: No module named 'nbconvert.exporters.webpdf'
[Errno 2] No such file or directory: 'first-pipeline'
/content
total 4
drwxr-xr-x 1 root root 4096 May  3 13:42 sample_data
```

ploomber, version 0.18.1

**Comment by edublancas** [comment]
I saw something like this the other day. can you check which version of nbconvert is running? I saw a similar error with version 6.1.0. I think the solution is to check what minimum version we support and pin it

**Comment by idomic** [comment]
The version is nbconvert==5.6.1, on 6.5.0 it works but then it requires an update to jinja as well.

**Comment by edublancas** [comment]
alright, I fixed it so ploomber works with nbconvert 5.6.1 (and older versions)
